src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/AiaTests.cs

```diff
@@ -65,6 +64,7 @@ public static void DisableAiaOptionWorks()
                 cuCaStore.Open(OpenFlags.ReadWrite);
 
                 X509Chain chain = holder.Chain;
+                chain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
```

Comment:
This is already set for macOS down on line 108.  That chunk should be removed, and perhaps this one can be conditional?

Reply:
@bartonjs Oops. Sorry about that. Fixed (hopefully).